conf/application.conf

```diff
@@ -144,3 +147,37 @@ hec-tax-check {
   expires-after = 120 days
   ttl           = 150 days
 }
+
+hec-file-extraction-details {
+   extraction-timezone   = "Europe/London"
+   extraction-time       = "13:02"
```

Comment:
by default we're supposed to be doing it around midnight - could we have `00:00` as the default time here?